README.md

```diff
@@ -133,6 +135,28 @@ $stack->push(
 );
 ```
 
+for Flysystem 2
```

Comment:
Either this should be its own `##` heading or the Flysystem 1 version should also get this 'for xxx' heading.

The newer version should be before the older version

If keeping 'for' headings, should be capitalizd.

Reply:
Just added that.